sale_double_validation/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    'name': 'Double validation for Sales',
+    'summary': "",
+    'version': '12.0.1.0.0',
+    'author': 'Camptocamp,Odoo Community Association (OCA)',
+    'maintainer': 'Camptocamp',
+    'license': 'AGPL-3',
+    'category': 'sale',
+    'depends': [
+        'sale',
```

Comment:
@tschanzt no need to `sale`,  `sale_management` depends on `sale`

Reply:
@nikul-serpentcs you're right. IMHO, I think explicit is always better than implicit :smile:

Reply:
Hello @nikul-serpentcs , with the answer from @damdam-s , is it ok for you?
If yes can you update your review?